gh-docs/developer-guide/integration-studio-catalogue.md

```diff
@@ -150,6 +150,33 @@ Integration Studio has been released with the following package components.
 
 3. Clone Integration Studio GitHub repository https://github.com/wso2/integration-studio.git. and proceed with **main** branch.
 
+---
+**If you are using maven version 3.8.1 or higher**
+
+From maven version 3.8.1, maven default settings file updated with http blocker mirror. To prevent maven from blocking the http transfer we can 
+
+- Option 1 (not recommended) : Go to your $MAVEN_HOME/conf/settings.xml file and comment out mirror with id : "maven-default-http-blocker". This is a global change and will affect all of you maven project. 
+- Option 2 (recommended) : We can add project level maven settings file to disble http blocker for our wso2 repository. 
```

Comment:
This should start as you can add, as we're using go to your ... in the previous option. Both should be in same mood.